src/lib/components/LandingPage.svelte

```
@@ -29,54 +26,77 @@
-			>
-				{@html testDetails.start_instructions}
-			</p>
+			<div class="border-border mb-24 overflow-hidden rounded-2xl border bg-white">
+				<div class="bg-section-header border-border flex h-16 items-center gap-8 border-b px-5">
+					<span class="text-muted-foreground text-xs font-bold tracking-wider uppercase"
+						>{$t('Test Instructions')}</span
+					>
+				</div>
+				<div class="px-5 py-4">
+					<div class="text-foreground prose prose-sm text-[13px]/relaxed">
+						{@html testDetails.start_instructions}
```

ESLint

[error] `{`@html`}` can lead to XSS attack.

(svelte/no-at-html-tags)